pkgs/by-name/ol/ollama/package.nix

```
@@ -180,10 +181,15 @@ goBuild {
   );
 
   preBuild = ''
-    # disable uses of `git`, since nix removes the git directory
-    export OLLAMA_SKIP_PATCHING=true
     # build llama.cpp libraries for ollama
-    go generate ./...
+    make -j 5
```

This should probably be changed to be proportional to the available cores.

```suggestion
    make -j "$(nproc)"
```

It may be better to use fewer than the total thread count.

```suggestion
    threads="$(nproc)"
    make -j $((threads / 2))
```

---

$NIX_BUILD_CORES

---

don't know such var we can use.  $NIX_BUILD_CORES looks good!